custom_model_runner/requirements.txt

```diff
@@ -6,7 +6,7 @@ jinja2
 memory_profiler<1.0.0
 mlpiper~=2.4.0
 numpy
-pandas
+pandas>=1.2.5,<1.3.1
```

Comment:
how come you didnt want to pin?

Reply:
Doing this upper/lower constraint approach is nicer and is clearer distinction of the specific upper constraint that we _do not want_ (i.e. we cannot use `pandas==1.3.1`)